spec/lib/tmuxinator/config_spec.rb

```diff
@@ -15,7 +15,11 @@
 
   describe "#default" do
     it "gets the path of the default config" do
-      expect(Tmuxinator::Config.default).to include("default.yml")
+      allow(Tmuxinator::Config).to receive(:exists?).and_return(true)
+      expect(Tmuxinator::Config.default("default")).to include("default.yml")
+    end
+    it "maps the path of sample if default fails" do
```

Comment:
Again, for consistency, there should be an empty line above this one (in between `end` and `it`.